test-client.py

```diff
@@ -2,12 +2,12 @@
 '''Listen for recceiver Announcements
 '''
 
-import sys, socket
+import socket
 
 S = socket.socket(socket.AF_INET, socket.SOCK_DGRAM, 0)
 
 S.bind(('',5049))
 
-print S, S.fileno()
+print(S, S.fileno())
```

Comment:
Wait, is this python2 or python3 code? Were these tests being run?

Reply:
I'm guessing python2 code. I think this is more code to create a quick and dirty python reccaster. Could just remove?